build_tools/cmake/iree_macros.cmake

```diff
@@ -514,6 +514,37 @@ function(iree_symlink_tool)
   )
 endfunction()
 
+#-------------------------------------------------------------------------------
+# Test timeout values
+#-------------------------------------------------------------------------------
+
+# iree_get_timeout_seconds()
+#
+# Evaluates a Bazel-like timeout string to a number of seconds.
+# Allowed timeout values:  "short", "moderate", "long".
+# Default is "short".
+#
+# This is aligned with our `native_test` function in Bazel. This differs from
+# general Bazel rules in two ways: they default to "moderate" and they allow an
+# additional timeout value "eternal".
+#
+# Note: the current numbers of seconds are just the Bazel standard values.
```

Comment:
Could link to https://bazel.build/reference/test-encyclopedia#role-test-runner here